src/java/picard/illumina/CollectIlluminaBasecallingMetrics.java

```diff
@@ -43,25 +43,49 @@
 import java.util.TreeMap;
 
 /***
- *  A Command line tool to collect Illumina Basecalling metrics for a sequencing run
- *  Requires a Lane and an input file of Barcodes to expect.
- *  Outputs metrics:
- *    *  Mean Clusters Per Tile
- *    *  Standard Deviation of Clusters Per Tile
- *    *  Mean Pf Clusters Per Tile
- *    *  Standard Deviation of Pf Clusters Per Tile
- *    *  Mean Percentage of Pf Clusters Per Tile
- *    *  Standard Deviation of Percentage of Pf Clusters Per Tile
- */
+        - *  A Command line tool to collect Illumina Basecalling metrics for a sequencing run
+        - *  Requires a Lane and an input file of Barcodes to expect.
+        - *  Outputs metrics:
+        - *    *  Mean Clusters Per Tile
+        - *    *  Standard Deviation of Clusters Per Tile
+        - *    *  Mean Pf Clusters Per Tile
+        - *    *  Standard Deviation of Pf Clusters Per Tile
+        - *    *  Mean Percentage of Pf Clusters Per Tile
+        - *    *  Standard Deviation of Percentage of Pf Clusters Per Tile
+        - */
+
 @CommandLineProgramProperties(
-        usage = CollectIlluminaBasecallingMetrics.USAGE,
-        usageShort = CollectIlluminaBasecallingMetrics.USAGE,
+        usage = CollectIlluminaBasecallingMetrics.USAGE_SUMMARY + CollectIlluminaBasecallingMetrics.USAGE_DETAILS,
+        usageShort = CollectIlluminaBasecallingMetrics.USAGE_SUMMARY,
         programGroup = Illumina.class
 )
 public class CollectIlluminaBasecallingMetrics extends CommandLineProgram {
+    static final String USAGE_SUMMARY = "Collects Illumina Basecalling metrics for a sequencing run.  ";
+    static final String USAGE_DETAILS = "<p>Program produces per-barcode and per-lane basecall metrics for each sequencing run.  Mean values for each metric are determined using data from all of the tiles.  This tool requires the LANE(#), BASECALLS_DIR, READ_STRUCTURE, and an input file listing the sample barcodes into the program.  " +
+            "Program will provide metrics including:" +
+            " the total numbers of bases, reads, and clusters, as well as the fractions of each bases, reads, and clusters that passed " +
+            "Illumina quality filters (Pf) both per barcode and per lane.  " +
+            "For additional information on Illumina's Pf quality metric, please see the corresponding " +
+            "<a href='https://www.broadinstitute.org/gatk/guide/article?id=6329'>GATK Dictionary entry</a>.</p> " +
+            "<p>The input barcode_list.txt file is a file containing all of the sample and molecular barcodes and can be obtained from the <a href='http://broadinstitute.github.io/picard/command-line-overview.html#ExtractIlluminaBarcodes'>ExtractIlluminaBarcodes</a> tool.  "   +
+
+            "<h4>Usage example:</h4>" +
+            "<pre>" +
+            "java -jar picard.jar CollectIlluminaBasecallingMetrics\\<br />" +
+            "      BASECALLS_DIR=s_1_1101.bcl \\<br />" +
```

Comment:
So the BASECALLS_DIR is just a directory name to put the files, right?  Thus, it doesn't need to have any special designation and can be called, e.g. /User/baseCall_directory/?